src/ThreeEditor/components/Editor/EditorAppBar/EditorAppBar.tsx

```diff
@@ -116,30 +117,35 @@ function EditorAppBar({ editor }: AppBarProps) {
 			[
 				{
 					label: 'New',
+					id: 'newButton',
 					icon: <FiberNewIcon />,
 					disabled: false,
 					onClick: () => setNewProjectDialogOpen(true)
 				},
 				{
 					label: 'Open',
+					id: 'openButton',
 					icon: <FolderOpenIcon />,
 					disabled: false,
 					onClick: () => setOpenFileDialogOpen(true)
 				},
 				{
 					label: 'Undo (ctrl+z)',
+					id: 'undoButton',
 					icon: <UndoIcon />,
 					disabled: !canUndo,
 					onClick: () => editor?.history.undo()
 				},
 				{
 					label: 'Save as',
+					id: 'saveAsButton',
 					icon: <SaveAsIcon />,
 					disabled: false,
 					onClick: () => setSaveFileDialogOpen(true)
 				},
 				{
 					label: 'Redo (ctrl+y)',
+					id: 'redoButton',
```

Comment:
Shouldn't these be reverted or changed to aria?

Probably aria-label enough, or maybe we should use aria menu? 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role#examples

Reply:
yes, these were unnecessary